iceoryx_posh/cmake/iceoryx_posh_deployment.cmake

```diff
@@ -0,0 +1,34 @@
+# configure deployment
+if(NOT IOX_MAX_PORT_NUMBER)
+    set(IOX_MAX_PORT_NUMBER 1024u)
+endif()
+
+if(NOT IOX_MAX_INTERFACE_NUMBER)
+    set(IOX_MAX_INTERFACE_NUMBER 4u)
+endif()
+
+if(NOT IOX_MAX_RECEIVERS_PER_SENDERPORT)
+    set(IOX_MAX_RECEIVERS_PER_SENDERPORT 256u)
+endif()
+
+if(NOT IOX_MAX_SUBSCRIBERS_PER_PUBLISHER)
```

Comment:
Just curious, are receiver ports and subscribers not one-to-one ? These two options seem redundant.